modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/thunks/fieldChange.es.js

```diff
@@ -10,6 +10,20 @@ import {disableSubmitButton} from '../utils/submitButtonController.es';
 
 let REVALIDATE_UPDATES = [];
 
+const requireUpdatePageFieldNames = [
+	'autocomplete',
+	'inputMask',
+	'numericInputMask',
+	'options',
+	'requireConfirmation',
+	'required',
+	'validation',
+];
```

Comment:
Hi @IstvanD. According to the previous ci tests I ran on the [initial PR](https://github.com/alinedoleron/liferay-portal/pull/256), some tests have failed. I fixed most of them and I hope I have covered the main ones.

So please, add the following missing fields:

```suggestion
const requireUpdatePageFieldNames = [
	'alphabeticalOrder',
	'autocomplete',
	'dataSourceType',
	'ddmDataProviderInstanceId',
	'expirationDate',
	'hideField',
	'inputMask',
	'limitToOneSubmissionPerUser',
	'multiple',
	'neverExpire',
	'numericInputMask',
	'options',
	'predefinedValue',
	'requireConfirmation',
	'required',
	'sendEmailNotification',
	'validation',
];
```

Sorry for the inconvenience. 
Regards,

Reply:
hi @alinedoleron thank you for your feedback. I have added the missing fields.
https://github.com/liferay-forms/liferay-portal/pull/2400/commits/537b54f88435b5bfa91a614e6af4225852a4c730